Src/Web/Web.Shared.Net.Tests/RequestTrackingTelemetryModuleTest.cs

```diff
@@ -495,10 +521,10 @@ private string GetActivityRootId(string telemetryId)
             return telemetryId.Substring(1, telemetryId.IndexOf('.') - 1);
         }
 
-        private RequestTrackingTelemetryModule RequestTrackingTelemetryModuleFactory(TelemetryConfiguration config = null)
+        private RequestTrackingTelemetryModule RequestTrackingTelemetryModuleFactory(TelemetryConfiguration config = null, CorrelationIdLookupHelper correlationHelper = null)
```

Comment:
did you expose this override dictionary as a public property on telemetry modules? THis is how people will override it